interfaces/python/CMakeLists.txt

```diff
@@ -267,13 +267,26 @@ install(
 # Standalone interface builds for wheels will need the files to be copied to a different location anyway,
 # so not copying the files for now isn't a big loss
 if(WIN32)
+    # Realistically for bundling into a Python wheel, the dll should be given a unique name
+    # to avoid conflicts with other packages using a different version of the dll with the same name
```

Comment:
So this isn't changing the name yet is it?  looks like just copying the file in a more robust way

Reply:
No, it isn't changing the name. A name conflict is more likely to be a problem with the libzmq library than helicsSharedLib (more Python projects could be using something with a copy of ZeroMQ).

I think the name mangling should probably be handled by a step when generating the wheels that goes through and bundles any needed shared libraries on the system PATH into the generated wheel and gives them unique names.